_posts/2020-07-27-bikesharing-demand/2020-07-27-bikesharing-demand.Rmd

```diff
@@ -0,0 +1,1100 @@
+---
+title: Bike Sharing Demand - Use Case
+author: 
+  - name: Henri Funk
+  - name: Andreas Hofheinz
+  - name: Marius Girnat
+date: 07-27-2020
+description: |
+  This use case provides an example on tuning and benchmarking in mlr3verse using data from Capital Bikeshare.
+categories:
+  - 'tuning'
+  - 'benchmarking'
+  - 'nested resampling'
+  - 'filter'
+  - 'branching'
+bibliography: bs.bib
+output:
+  distill::distill_article:
+    self_contained: false
+    css: ../../custom.css
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(
+  echo = TRUE,
+  R.options = list(width = 80)
+)
+library(mlr3book)
+```
```

Comment:
Create a comment and explain the `long_run` thing, say this needs to be changed here.